src/main/java/uk/gov/companieshouse/api/accounts/validation/DebtorsValidator.java

```diff
@@ -160,9 +152,9 @@ private boolean validateNoUnexpectedDataPresent(boolean hasCurrentPeriodBalanceS
     }
 
     private void validateCurrentPeriodFields(CurrentPeriod debtorsCurrentPeriod, Errors errors) {
-        if (debtorsCurrentPeriod.getTotal() == null) {
+        if (debtorsCurrentPeriod.getTotal() == null && debtorsCurrentPeriod.getDetails() == null) {
```

Comment:
Should we do a null or empty check, else it'll be possible to submit a current period with additional info set to `""`? Maybe use apache commons' `StringUtils#isBlank`?

Reply:
As suggested. Changed to use the isBlank method to guard against white space and empty submissions. Thanks!